app/models/request.rb

```diff
@@ -30,6 +31,14 @@ def current_stage
     stages.find_by(:state => [Stage::NOTIFIED_STATE, Stage::PENDING_STATE])
   end
 
+  def num_children
```

Comment:
Where does `num_children` get called from and we really should fully spell out the name.  `number_of_children` or `child_size`.

Reply:
They are replacing `total_stages` and `current_active_stage`. They are included in openapi spec for UI to quickly display the overall progress.